### PR TITLE
Update plugins

### PR DESCRIPTION
**Pull request by KilianPoirier**

### CODE REVIEW

File: docs/install-openqaoa.md [hunk 1]
```diff
@@ -10,6 +10,23 @@ Once the virtual environment is activated, then simply pip install openqaoa with
 pip install openqaoa
 ```
 
+!!! note
+    This command installs the whole openqaoa package including all available plugins
+
+OpenQAOA is now divided into plugins to prevent installing unnecessary modules. Each plugin is available through pip and allows you to access the different providers and backends. Currently, OpenQAOA counts 5 different plugins:
```
Comment (vishal-ph) [inline docs/install-openqaoa.md:16]: shall we rephrase it to something more intentional like, "OpenQAOA is now divided into plugins to let users selectively install modules corresponding to their needs", or something similar?

Comment (KilianPoirier) [reply docs/install-openqaoa.md:16]: Yes, I think your sentence shows the intention of the plugins much better.

**Review by vishal-ph — APPROVED** [review]

**Comment by vishal-ph** [comment]
I have approved the PR, but did you want to merge directly to `main` instead of `dev`?
